book/06-github/sections/3-maintaining.asc

```diff
@@ -1,134 +1,133 @@
-Users will have to have an account and an uploaded SSH key to access your project if you give them the SSH URL.
-The HTTPS one is also exactly the same URL they would paste into a browser to view the project there.
+Madalas na lalong kanais-nais na ibahagi ang URL na nakabatay sa HTTPS para sa isang pampublikong proyekto, dahil ang gumagamit ay hindi kailangang magkaroon ng isang account sa GitHub upang ma-access ito para sa pag-clone.
+Ang mga gumagamit ay magkakaroon ng isang account at isang na-upload na SSH key upang ma-access ang iyong proyekto kung binibigyan mo sila ng SSH URL.
+Ang HTTPS ay isa ring eksaktong parehong URL na kanilang idikit sa isang browser upang tingnan ang proyekto doon.
 ====
 
-==== Adding Collaborators
+==== Pagdaragdag ng mga Tagapangasiwa
 
-If you're working with other people who you want to give commit access to, you need to add them as ``collaborators''.
-If Ben, Jeff, and Louise all sign up for accounts on GitHub, and you want to give them push access to your repository, you can add them to your project.
-Doing so will give them ``push'' access, which means they have both read and write access to the project and Git repository.
+Kung ikaw ay nagtatrabaho kasama ang ibang tao na nais mong bigyan ng access sa pag-commit, kailangan mong idagdag sila bilang ``tagapangasiwa''.
+Kung si Ben, Jeff, and Louise ay nag-sign up ng mga account sa GitHub, at gusto mo silang bigyan ng access sa pag-push sa iyong repositoryo, maaari mo silang idagdag sa iyong proyekto.
+Ang paggawa nito ay nagbibigay sa kanila ng access sa ``push'', na nangangahulugan na mayroon silang access sa pagbasa at pagsulat sa proyekto at repositoryo ng Git.
 
-Click the ``Settings'' link at the bottom of the right-hand sidebar.
+I-click ang link ng ``Settings'' sa ibaba ng kanang sidebar.
 
-.The repository settings link.
-image::images/reposettingslink.png[The repository settings link.]
+.Ang link ng settings ng repositoryo.
+image::images/reposettingslink.png[Ang link ng settings ng repositoryo.]
 
-Then select ``Collaborators'' from the menu on the left-hand side.
-Then, just type a username into the box, and click ``Add collaborator.''
-You can repeat this as many times as you like to grant access to everyone you like.
-If you need to revoke access, just click the ``X'' on the right-hand side of their row.
+Pagkatapos piliin ang =``Tagapangasiwa'' mula sa menu sa kaliwang bahagi.
+Pagkatapos, magtipa ng username sa kahon, at i-click ang ``Magdagdag ng tagapangasiwa.''
+Maaari mong ulitin ito nang maraming beses hangga't gusto mo upang magbigay ng access sa lahat ng iyong gusto
+Kung kailangan mong bawiin ang access, i-click lamang ang `` X '' sa kanang bahagi ng kanilang hilera.
 
-.Repository collaborators.
-image::images/collaborators.png[The repository collaborators box.]
+.Tagapangasiwa ng Repositoryo.
+image::images/tagapangasiwa.png[Ang kahon ng tagapangasiwa ng repositoryo.]
```

Comment:
@heywens was the translation of the image file really intended? This `.png` does not exist...